.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: objective-c
+os: osx
+osx_image:
+  - xcode11.3
+  - xcode11.2
+  - xcode10.3
+before_install:
+  - wget -O bazelisk "https://github.com/bazelbuild/bazelisk/releases/download/v1.2.1/bazelisk-darwin-amd64"
+  - chmod +x "bazelisk"
```

Comment:
Should the checksum be validated after download?

Reply:
Added a check.

Reply:
Thanks!